Source/Scene/SkyAtmosphere.js

```diff
@@ -161,18 +188,38 @@ SkyAtmosphere.prototype.update = function (frameState) {
     return undefined;
   }
 
+  // Align the ellipsoid geometry so it always faces the same direction as the
+  // camera to reduce artifacts when rendering atmosphere per-vertex
+  var view = frameState.context.uniformState.viewRotation;
+  var inverseView = Matrix3.inverse(view, scratchRotationMatrix);
```

Comment:
can use `uniformState.inverseViewRotation` instead, or `frameState.camera.inverseViewMatrix` (and set the translation to zero after) to avoid going through `UniformState`.